File: ChangeScreenByJava/app/src/main/java/com/example/changescreenbyjava/MainActivity.java
package com.example.changescreenbyjava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void Login(View view) {

        Intent intent = new Intent(MainActivity.this, Login.class);
        startActivity(intent);
        setResult(RESULT_OK, intent);
    }
}

File: ChangeScreen/app/src/main/java/com/example/changescreen/MainActivity.kt
package com.example.changescreen

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatViewInflater

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun login(view: View){
        val changePage = Intent(this, )

        startActivity(changePage)
    }
}
